static/css/timetable/partials/course_modal.scss

```diff
@@ -65,6 +65,97 @@
   border-top-left-radius: 5px;
 }
 
+.areas-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: flex-start;
+  align-items: center;
+  align-content: flex-start;
+  height: 25px;
+  margin: 6px;
+  &.search {
+    height: 20px;
+    margin: 0px;
+  }
+}
+
+.course-and-dept-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: flex-start;
+  align-items: center;
+  align-content: flex-start;
+  height: 21px;
+  margin: 0px 7px 0px 0px;
+}
+
+.area-bubble {
+  background: blue;
+  border-radius: 25%;
+  color: white;
+  text-align: center;
+  vertical-align: middle;
+  font-family: $font-body;
+  font-size: 17px;
+  font-weight: bold;
+  height: 21px;
+  width: 21px;
+  padding: 1px 2.5px 0px 2px;
+  margin: 2px;
+  text-align: center;
+  &.search {
+    font-size: 15px;
+    height: 20px;
+    width: 20px;
+    margin: 4px;
+    padding: 2px;
+  }
+  &.H {
```

Comment:
try taking this out of being a child of area-bubble so that you don't need to repeat it later